CRM/Admin/Page/APIExplorer.php

```diff
@@ -105,7 +105,9 @@ public static function getExampleFile() {
    * Ajax callback to display code docs
    */
   public static function getDoc() {
-    if (!empty($_GET['entity']) && strpos($_GET['entity'], '.') === FALSE) {
+    // Verify the API handler we're talking to is valid.
+    $entities = civicrm_api3('Entity', 'get');
+    if (!empty($_GET['entity']) && in_array($_GET['entity'], $entities['values']) && strpos($_GET['entity'], '.') === FALSE) {
       $entity = _civicrm_api_get_camel_name($_GET['entity']);
```

Comment:
This line can be deleted now, since we verified above that it's already in CamelCase.